File: app/src/main/java/com/agrogames/islandsofwar/common/Arr.java
package com.agrogames.islandsofwar.common;

public class Arr {
    public static int indexOf(int[] array, int item){
        for (int i = 0, arrayLength = array.length; i < arrayLength; i++) {
            int at = array[i];
            if (at == item) return i;
        }
        return  -1;
    }
}

File: app/src/main/java/com/agrogames/islandsofwar/common/Arr.kt
package com.agrogames.islandsofwar.common

object Arr {
    fun indexOf(array: IntArray, item: Int): Int {
        var i = 0
        val arrayLength = array.size
        while (i < arrayLength) {
            val at = array[i]
            if (at == item) return i
            i++
        }
        return -1
    }
}